spec/lib/tasks/order_cycles_rake_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require 'rake'
+
+describe 'order_cycles.rake' do
+  describe ':remove_order_cycle' do
+    before do
+      Rake.application.rake_require 'tasks/order_cycles'
+      Rake::Task.define_task(:environment)
+      Rake::Task["ofn:remove_order_cycle"].reenable
+    end
+
+    context 'when the order cycle exists' do
+      let!(:order_cycle) { create(:order_cycle) }
+
+      it 'removes the specified order cycle' do
+        expect {
+          Rake.application.invoke_task "ofn:remove_order_cycle[#{order_cycle.id}]"
+        }.to change(OrderCycle, :count).by(-1)
+      end
+
+      it 'removes the order cycle coordinator fees' do
+        expect {
+          Rake.application.invoke_task "ofn:remove_order_cycle[#{order_cycle.id}]"
+        }.to change(CoordinatorFee, :count).by(-1)
+      end
```

Comment:
I think you need to check exchanges are gone as well as exchange_variants

Reply:
Good point. I'll check that however none of this failed due to DB constraints. I managed to run this in production.

Reply:
yeah, but is the data gone from these tables? I would just triple check we dont end up with stale inconsistent data that can/will break the app in some random part.